services/plugins/generic_plugins/main.go

```diff
@@ -0,0 +1,17 @@
+package generic_plugins
+
+import (
+	"fmt"
+
+	"github.com/leanspace/terraform-provider-leanspace/provider"
+)
+
+var GenericPluginDataType = provider.DataSourceType[GenericPlugin, *GenericPlugin]{
+	ResourceIdentifier: "leanspace_generic_plugins",
+	Path:               "plugins-repository/generic-plugins",
+	Schema:             genericPluginSchema,
+	FilterSchema:       dataSourceFilterSchema,
+	ReadPath: func(id string) string {
+		return fmt.Sprintf("plugins-repository/generic-plugins/%s", id)
```

Comment:
The read operation is not different from what we have on other service no ? So I guess you can remove this

Reply:
Ok removed